examples/grid/grid_status_download_example.py

```diff
@@ -0,0 +1,59 @@
+# Example version of the gridstatus_download.py script
+# Download 2 days of data for the OKGE.FRONTIER location
+# Download both real time and day ahead data
```

Comment:
Somewhere, a short overview of the two example scripts (this one and "process_results.py") would be helpful. For example, does this script download the raw files and then "process_results.py" puts them in the format required by hercules?